pydough/configs/pydough_configs.py

```diff
@@ -65,6 +65,19 @@ class PyDoughConfigs:
     is False.
     """
 
+    collation_default_asc = ConfigProperty[bool](True)
+    """
+    If True, then the collation will default to `ASC`. If False, then the
+    collation will default to `DESC`. The default is True.
+    """
+
+    propogate_collation = ConfigProperty[bool](False)
+    """
+    If True, then the collation(`ASC` or `DESC`), if exists, will be propogated across
+    the nodes for that term. If False, then the collation will not be propogated across
```

Comment:
Clarify how the value will be identified
e.g. It will use the default collation as specified by `collation_default_asc`